code/game/machinery/computer/camera.dm

```diff
@@ -9,7 +9,7 @@
 
 	var/mapping = 0 // For the overview file (overview.dm), not used on this page
 
-	var/list/network = list()
+	var/list/network = list("SS13","Mining Outpost")
```

Comment:
Unsure if this is the best way to go about it, I would say to keep the network list in the circuit board itself, but there's no easy way to do it...

Reply:
Although that sounds like it would be something. What i understand from the different circuit boards is that they act like a "Build path" for the Computer frame. 
![Skärmbild 2021-08-21 220457](https://user-images.githubusercontent.com/72948218/130333496-ae802882-0482-4119-998d-84f79ba20058.png)

Add a security camera console(camera monitor) board and the computer frame is directed to that dm. Add a supermatter monitoring console board and the computer frame is directed to that dm. 

So the only way to change anything is the dm that is the final product. After looking through the code this is the most logical solution i could come up with.

But i wont be suprised if im dead wrong.